data/encryptVideos.js

```diff
@@ -23,11 +23,21 @@ module.exports = [
         "direct": "https://d24kjznqej0s8a.cloudfront.net/2016/encryption_campaign/beat4_landingpage",
         "social": "https://d24kjznqej0s8a.cloudfront.net/2016/encryption_campaign/beat4_landingpage",
         "title": "Encryption, Journalism, and Free Expression",
-        "description": "Encryption doesn't just protect your privacy -- it also enables free expression. Encryption helps shield journalists and their sources, allowing individuals to tell powerful stories.",
+        "description": "Encryption doesn't just protect your privacy — it also enables free expression. Encryption helps shield journalists and their sources, allowing individuals to tell powerful stories.",
```

Comment:
@alanmoo should we turn this em dash into unicode as well? Not sure what is preferred here but I see you used unicode below on line 39: `"cta": "Thanks for signing the pledge! Can\u00A0you\u00A0help us spread the word?"`